gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadChannel.java

```diff
@@ -813,8 +790,8 @@ protected void initMetadata(@Nullable String encoding, long sizeFromMetadata, lo
     metadataInitialized = true;
 
     logger.atFiner().log(
-        "Initialized metadata (gzipEncoded=%s, size=%s, randomAccess=%s, generation=%s) for '%s'",
```

Comment:
May be we can have a toString for the AdaptiveAccessPattern and have it logged here?